libs/types/src/tokens.rs

```diff
@@ -407,6 +365,76 @@ pub mod before {
 	}
 }
 
+pub mod usdc {
+	use sp_std::vec::Vec;
+
+	use super::*;
+
+	pub const MIN_SWAP_ORDER_AMOUNT_USDC: Balance = 10_000_000;
+	pub const DECIMALS_USDC: u32 = 6;
+	pub const EXISTENTIAL_DEPOSIT_USDC: Balance = 1000;
+
+	pub const CURRENCY_ID_DOT_NATIVE_USDC: CurrencyId = CurrencyId::ForeignAsset(6);
+	pub const CURRENCY_ID_LP_ETH_USDC: CurrencyId = CurrencyId::ForeignAsset(100_001);
+	pub const CURRENCY_ID_LP_ETH_GOERLI_USDC: CurrencyId = CurrencyId::ForeignAsset(100_001);
+	pub const CURRENCY_ID_LP_BASE_USDC: CurrencyId = CurrencyId::ForeignAsset(100_002);
+	pub const CURRENCY_ID_LP_ARB_USDC: CurrencyId = CurrencyId::ForeignAsset(100_004);
+	pub const CURRENCY_ID_LP_CELO_USDC: CurrencyId = CurrencyId::ForeignAsset(100_005);
```

Comment:
NIT and not for this PR at all. But having them now inside a usdc module, maybe we could strip the suffix `_USDC`, the user would only import `usdc` having access to all constants. i.e: `usdc::CURRENCY_ID_DOT_NATIVE`.

Reply:
I was thinking the same. Will do!